src/on.ts

```diff
@@ -104,3 +104,58 @@ export default function on(target: any, type: any, listener: any, capture?: bool
 
 	throw new TypeError('Unknown event emitter object');
 }
+
+/**
+ * Provides a mechanism for listening to the next occurence of events from event
+ * emitters, Evented objects, or DOM nodes.
+ * @param target Target to listen for event on
+ * @param type Event type(s) to listen for; may be strings or extension events
```

Comment:
I realize now that I neglected this before when I cleared out extension events, but we should remove mention of them from jsdoc comments.